packages/mc-scripts/package.json

```diff
@@ -42,7 +42,7 @@
     "@commercetools-frontend/mc-html-template": "16.5.3",
     "@svgr/webpack": "5.2.0",
     "babel-loader": "8.1.0",
-    "babel-plugin-react-intl": "6.0.1",
+    "babel-plugin-react-intl": "7.1.0",
```

Comment:
> babel-plugin-react-intl: As of react-intl@4.x, FormattedHTMLMessage is no longer supported so
we're removing this from our babel plugin as well

I've not seen us or anybody using that. In theory it would be a breaking change also for the mc-scripts. I think however mentioning it on the release notes would be enough?